test/unit/lib/plugins/aws/deployFunction.test.js

```diff
@@ -417,10 +417,19 @@ describe('AwsDeployFunction', () => {
           },
           Timeout: 12,
           MemorySize: 512,
-          Role: 'arn:aws:us-east-1:123456789012:role/role',
+          Role: 'my-role',
         })
       ).to.be.equal(true);
     });
+
+    it('should prefer provider.iam.role over provider.role config', async () => {
```

Comment:
I know it adds a bit of extra work here, but I think this test case should be written using `runServerless` as we're trying to move from these type of tests altogether.

Reply:
I tried to find an example, however couldn't find one. I assumed it's due to the fact it's about deployment and stuff? Happy to change though

Reply:
@runk all testing related documentation can be found here: https://github.com/serverless/serverless/tree/master/test#how-to-run-organize-and-write-tests